File: 프로그래머스/0/181949. 대소문자 바꿔서 출력하기/대소문자 바꿔서 출력하기.cpp
#include <iostream>
#include <string>

using namespace std;

int main(void) {
    string str;
    int diff = 'a' - 'A';
    cin >> str;
    for(auto c: str)
        cout << ((c >= 'A' && c <= 'Z') ? char(c+diff) : char(c-diff));
    return 0;
}

File: 프로그래머스/0/181949. 대소문자 바꿔서 출력하기/대소문자 바꿔서 출력하기.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String a = sc.next();
        for(int i = 0; i < a.length(); i++) {
            char c = a.charAt(i);
            int diff = 'a' - 'A';
            if(c >= 'a' && c <= 'z') {
                System.out.print((char)(c - diff));
            } else {
                System.out.print((char)(c + diff));
            }
        }
    }
}